solutions/HE Twin Permutations.mdx

```diff
@@ -0,0 +1,70 @@
+---
+id: HE-twin-permutations
+source: HackerEarth
+title: Twin Permutations
+author: Andrew Wang
+---
+<Spoiler title =  "Solution">
+
+**Time Complexity:** $\mathcal{O}(N\log{N})$
+
+We can imagine the permutations as a graph where the $x$-coordinates are the indices of the second array and the $y$-coordinates are the indices of the first array 
```

Comment:
This sentence is confusing. Why not - Let $x_i$ be the index of $i$ in the second array?